### PR TITLE
get rid of shared const (close #68)

### LINKED ISSUE #68: Get rid of shared const.
Split `const.js` into the semantically separated groups of constants and put them in the appropriate places. Remove shared const term from the code BTW. As an addition: rename `service-msg-cmd` to `commands` (like with `test-run` in testcafe package) and move it to the `/session` dir with the `session.js` and `cookie.js` (`session.js` being an `index.js` there).

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @inikulin @churkin

### CODE REVIEW

File: src/processing/dom/index.js [hunk 1]
```diff
@@ -152,6 +152,8 @@ export default class DomProcessor {
         var replacer = (resourceUrl, resourceType, charsetAttrValue) => urlReplacer(resourceUrl, resourceType, charsetAttrValue, baseUrl);
         var $all     = $('*');
 
+        var ELEMENT_IS_PROCESSED = 'hammerhead|element-is-processed';
```
Comment (inikulin) [inline src/processing/dom/index.js]: `const` here

File: src/client/dom-processor/client-dom-adapter.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ export default class ClientDomAdapter extends BaseDomAdapter {
     }
 
     isTopParentIframe (el) {
-        var elWindow = el[DOM_SANDBOX_PROCESSED_CONTEXT];
+        var elWindow = el[INTERNAL_ATTRS.processedContext];
```
Comment (inikulin) [inline src/client/dom-processor/client-dom-adapter.js]: Ok, as I can see, here we have property, not attribute. So, seems, like we need to split constants in two groups: internal properties and internal attributes. We can put them in `processing/dom/internal/properties.js' and `processing/dom/internal/attributes.js' respectively.

File: src/client/sandbox/node/index.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ export default class NodeSandbox extends SandboxBase {
         super();
 
         this.raiseBodyCreatedEvent       = this._onBodyCreated;
-        document[CONST.DOCUMENT_CHARSET] = parseDocumentCharset();
+        document[INTERNAL_ATTRS.documentCharset] = parseDocumentCharset();
```
Comment (inikulin) [inline src/client/sandbox/node/index.js]: Formatting is broken here

**Comment by inikulin** [comment]
\r-

File: src/processing/dom/index.js (hunk 1 above)
Comment (miherlosev) [inline src/processing/dom/index.js]: This place will be fixed in separate issue - https://github.com/superroma/testcafe-hammerhead/issues/194

File: src/client/dom-processor/client-dom-adapter.js (hunk 2 above)
Comment (miherlosev) [inline src/client/dom-processor/client-dom-adapter.js]: Internal properties:
    overrideDomMethodName:     'hammerhead|override-dom-method',
    processedContext:          'hammerhead|processed-context',
    whichPropertyWrapper:      'hammerhead|which-property-wrapper',
    documentCharset:           'hammerhead|document-charset',
Internal attributes:
    hoverPseudoClass:          'data-hammerhead-hovered',
    focusPseudoClass:          'data-hammerhead-focused',
    storedAttrPostfix:         '-hammerhead-stored-value',
    uploadInfoHiddenInputName: 'hammerhead|upload-info-hidden-input-name'

File: src/processing/dom/index.js (hunk 1 above)
Comment (inikulin) [inline src/processing/dom/index.js]: I don't see why it's a reason to not write good **new** code

File: src/client/dom-processor/client-dom-adapter.js (hunk 2 above)
Comment (inikulin) [inline src/client/dom-processor/client-dom-adapter.js]: Looks good

**Comment by miherlosev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
@miherlosev What was changed after my review?

**Comment by miherlosev** [comment]
as say @churkin 
we need to change order for `import` as follow:
at first - constants
at second - classes
then utils and separate methods (`import` as {...} from)

**Comment by inikulin** [comment]
@miherlosev Guys don't do this, let's not complicate rules. Currently the rule is simple, let's keep it:
at first external or built-in deps, then internal deps.

**Comment by churkin** [comment]
lgtm

**Pull request merged**